py-polars/polars/dataframe/frame.py

```diff
@@ -3432,6 +3433,12 @@ def write_parquet(
             The `partition_cols` parameter leads to write the dataset to a directory.
             Similar to Spark's partitioned datasets.
 
+        Attention
```

Comment:
This was very unexpected when working with multiple `write_parquet` calls in succession :eyes:

Reply:
I don't think this will be the behavior after what I said above.